odoo/addons/base/models/assetsbundle.py

```diff
@@ -7,12 +7,7 @@
 import json
 import textwrap
 import uuid
-try:
-    import sass as libsass
-except ImportError:
-    # If the `sass` python library isn't found, we fallback on the
-    # `sassc` executable in the path.
-    libsass = None
+import sass as libsass
```

Comment:
Also remove compiler support in ScssStylesheetAsset since it should not be necessary / used anymore.